.env_example

```diff
@@ -10,6 +10,4 @@ export APP_ENV=local
 
 export PORT=3000
 
-export SDR_USER=sdrUser
 export SDR_PASS=sdrPass
-
```

Comment:
Sorry, but I really want this file left alone; it's not obvious, but this is very useful to have this defined for local development and testing.  Leaving it as is has no impact on the code or deployment.